Firestore/Example/Tests/Local/FSTMutationQueueTests.mm

```diff
@@ -554,22 +529,19 @@ - (NSUInteger)batchCount {
 }
 
 /**
- * Removes entries from from the given @a batches and returns them.
+ * Removes the first n entries from the from the given batches and returns them.
```

Comment:
Nit: s/from the from the/from the

Reply:
I blame it on the original `from from`. Fixed :)